SampleApp/Podfile

```diff
@@ -1,6 +1,4 @@
 platform :ios, '12.0'
-inhibit_all_warnings!
-use_frameworks!
```

Comment:
This fixes a weird thing with Cocoapods I was seeing where the updated framework wasn't getting linked into the SampleApp, causing crashes. Couldn't debug any further so gave up and went with this...

Reply:
Suspect this is a problem with your local environment, and shouldn't need to be changed as part of this PR.

Reply:
I've seen this same problem across Blueprint and Listable, on 3 separate computers and OS versions (some on Catalina, some not) for several months. Not sure how this could be a local thing given that...

Reply:
(I believe `use_frameworks!` fixes the underlying issue – but we also shouldn't be hiding warnings, so also removed that.)